client/my-sites/plans/jetpack-plans/constants.ts

```diff
@@ -189,19 +176,44 @@ export const INDIRECT_CHECKOUT_PRODUCT_STATS = (): SelectorProduct => ( {
 	hidePrice: true,
 	// TODO: Refactor the checkout URL.
 	externalUrl: '/stats/purchase/{siteSlug}?from=calypso-plans',
-} );
 
-export const INDIRECT_CHECKOUT_PRODUCT_STATS_PWYW_YEARLY = (): SelectorProduct => ( {
-	...INDIRECT_CHECKOUT_PRODUCT_STATS(),
+	displayName: translate( 'Stats' ),
+	shortName: translate( 'Stats' ),
```

Comment:
I know we're using "Stats" because we want to show "Stats" in the plan card header, but it might be worth noting in a code comment that this is also referred to as "Stats (Personal)" in [other parts of the code base](https://github.com/Automattic/wp-calypso/pull/79852/files#diff-a4c2bc80667e433c750f7ee4480fe2db44d151c9d85019a78e2c600f050c960cR159).

Reply:
Yes, this was precisely what I was struggling with as well. Thanks for the suggestion for noting annotations. 👍🏼

Reply:
Commented in [7c5feeb](https://github.com/Automattic/wp-calypso/pull/79982/commits/7c5feebd5ad946ad7accfb5ed598968b42374f1d).